docs/usage.rst

```diff
@@ -21,7 +21,7 @@
                 level=0,
             )
 
-        return df.to_html(max_rows=6, max_cols=3)
+        return df.to_html(max_rows=6, max_cols=4)
```

Comment:
I guess this change is responsible for changing

![image](https://user-images.githubusercontent.com/173624/197141306-aac29923-409b-48de-ada4-2acdf5486526.png)

to

![image](https://user-images.githubusercontent.com/173624/197141366-1d047d40-f636-42bf-9ec8-08179967f9fe.png)

?

So maybe we switch to use `db.root` to make the `file` entry shorter as you suggested some time ago?

Reply:
Yes, I wanted to make sure that we see all columns in the multi-channel example:

![image](https://user-images.githubusercontent.com/10383417/197141713-ae666053-b015-4f49-a72c-67abfcc0da4a.png)

> So maybe we switch to use db.root to make the file entry shorter as you suggested some time ago?

I think we can achieve the same with the current solution, we simply remove a little more.

Reply:
I changed it to:

![image](https://user-images.githubusercontent.com/10383417/197143186-6e5cf3c3-69bf-4bb1-9191-0ed7ab819a06.png)

Reply:
Great, thanks.